### DISCUSSION TITLE
Provide the ability to control whether Insomnia Sync is allowed at the Organization level

### DISCUSSION THREAD

**Discussion by felderi** [Kong/insomnia#6694] [Ideas]
Organizations need the ability to control where their data in synchronized to assure the data in going only to approved backends. In many cases Organizations do not want to synchronize their collections to Insomnia cloud and force only GIt Sync or no sync at all to take place.

**Comment by subnetmarco** [comment]
This has been addressed with storage control https://konghq.com/blog/product-releases/insomnia-storage-control
